src/components/ProductCard.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+            <div class="border flex flex-col justify-center py-8 cursor-pointer" @click="displayDetail(product)">
+                <div class="flex justify-center items-center">
+                   <img :src="product.thumbnail" class="h-64 w-80">
+                </div>
+                <div class="px-10">
+                    <p>{{product.title}}</p>                            
+                    <p class="font-bold text-emerald-500">Rs.{{calculateDiscoutedAmount(product.price,product.discountPercentage)}}</p>
```

Comment:
imo instead of passing multiple parameters we can pass whole product as parameter and do somting like this

```
  calculateDiscoutedAmount(product){
          return (product.price-((product.price*product.discountPercentage)/100)).toFixed(2)
      },
```

Reply:
done